pjnat64/src/pjnat64/pj-nat64.c

```diff
@@ -0,0 +1,1248 @@
+
+#include <pj/types.h>
+
+#include <pjsip-ua/sip_inv.h>
+#include <pjsip/sip_types.h>
+
+#include <pjsua.h>
+
+#include <pjnat64/pj-nat64.h>
+
+#define THIS_FILE "pj_nat64.c"
+
+#define MODULE_VERSION "$Id: 497af2bf5811a519a87deb9821451887ec3200e6 $"
```

Comment:
I think this is not needed anymore, is not?

Reply:
@dshamaev-intermedia remove?

Reply:
yes, after we will merg it here, i think it's not relevant anymore

Reply:
Removed.